src/PollingBlockTracker.test.ts

```diff
@@ -874,49 +1130,6 @@ describe('PollingBlockTracker', () => {
           );
         });
 
-        it('should emit "latest" periodically afterward', async () => {
```

Comment:
this gets folded into `describe('after a block number is cached', () => {...}` below

Reply:
Good call!